src/components/ChatEntry.js

```diff
@@ -1,22 +1,48 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, onToggleLike}) => {
+
+  const messageColor = sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
+
+  const toggleLike = () => {
+    onToggleLike({
+      id: id,
+      sender: sender,
+      body: body,
+      timeStamp: timeStamp,
+      liked: !liked
+    });
+    console.log(`Updated id: ${id} message`)
```

Comment:
Be sure to remove print tests, and any comments that shouldn't be published before finalizing a project. 🧹😌